kolibri/core/assets/src/views/CorePage/AppBarPage.vue

```diff
@@ -44,6 +44,8 @@
       @cancel="languageModalShown = false"
     />
 
+    <MeteredConnectionNotificationModal />
```

Comment:
With the explicit trigger in mind, we also don't need to add this here. Just in places where a superuser initiates a download.

Reply:
@nucleogenesis let me know if it would be helpful for me to audit all of these locations